index.html

```diff
@@ -15,12 +15,28 @@
     <!-- navbar setup -->
     <header>
       <!-- phone navigation -->
+       <!-- Mobile Menu Toggle -->
+          <section class="menu-mobile showMenu">
+            <button class="close" type="button"><h2>X</h2></button>
```

Comment:
- [ ] The links are clickable even in desktop(you can't see them, but you can click them) _Check the video :arrow_double_down: _
- https://www.awesomescreenshot.com/video/15392101?key=eee8b9c0dc84b01e4f34b97565955a65

Reply:
- [ ] The menu should extend all the way 
![image](https://user-images.githubusercontent.com/62705947/223472700-ffa8702d-9c08-4d57-8b31-dbab05417cfd.png)

Reply:
Done